app/src/main/kotlin/com/stytch/androidexample/MainViewModel.kt

```diff
@@ -39,7 +39,7 @@ class MainViewModel : ViewModel() {
                     _authenticationError.value = null
                     _navigationState.value = NavigationState.Navigate("profile")
                 }
-                is StytchResult.Error -> _authenticationError.value = result.exception.reason.toString()
+                is StytchResult.Error -> _authenticationError.value = result.exception.toString()
```

Comment:
In these cases, instead of using `.toString()`, we can use `.message`, since it will always be an exception of type `StytchError`, which is guaranteed to have a message.